sdk/src/main/java/io/dapr/actors/runtime/AppToDaprHttpAsyncClient.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors.runtime;
+
+import io.dapr.actors.Constants;
+import io.dapr.actors.DaprClientBase;
+import okhttp3.*;
+import reactor.core.publisher.Mono;
+
+
+/**
+ * Http client to call Dapr's API for actors.
+ */
+//public class DaprHttpAsyncClient implements DaprAsyncClient {
+class AppToDaprHttpAsyncClient extends DaprClientBase implements AppToDaprAsyncClient {
```

Comment:
Same as above, just with the Http in the name you choose.

Reply:
Unless you feel strongly I'll keep the current name.  It can always be renamed.